website/source/docs/configuration/from-1.5/variables.html.md

```diff
@@ -278,3 +278,48 @@ precedence over earlier ones:
 
 ~> **Important:** Variables with map and object values behave the same way as
 other variables: the last value found overrides the previous values.
+
+### A variable value must be known :
```

Comment:
:100: :+1: :trophy: Nicely done on adding the variables table.

Reply:
thanks :D